.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
```

Comment:
Do you know if this includes `-beta` releases (and similar)?

Reply:
I think it should. It says the glob should match one or more characters [in the docs](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet), without any disqualifications. I debated between this pattern and mere `'*'`, but thought more explicit is better.